### DISCUSSION TITLE
关于代码学习

### DISCUSSION THREAD

**Discussion by pp517462915** [TanyaShue/MaaYYs#9] [Q&A]
你好，我最近在了解MmaFW的代码，发现没啥文档，刚好看见您的项目，有前端和后端，所以想去学习下，我想问下怎样入手python自动化的，以前是做java的，现在想弄下游戏脚本。期待您的答复，517462915@qq.com

**Comment by TanyaShue** [comment]
家里出事了，暂时没时间，可能还得几天

**Comment by TanyaShue** [comment]
到时候再联系你吧

**Comment by TanyaShue** [comment]
你加我qq吧 2289861427
